### PR TITLE
feat: deprecate ropsten and rinkeby

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Ropsten and RInkeby not supported anymore, so we deprecating these 2 test networks.

_On standby for 27 Oct_

## Changes

- Deprecate ropsten 
- Deprecate rinkeby

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/183540450

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ interface EtherscanNetworkApiDetails {
...
-const ropstenGsnRelayConfig = {
-  relayHub: "0x29e41C2b329fF4921d8AC654CEc909a0B575df20",
-  stakeManager: "0x762A4D5F51d8b2F9bA1B0412B45687cE0EfFD92B",
-  forwarder: "0x25CEd1955423BA34332Ec1B60154967750a0297D",
+const goerliGsnRelayConfig = {
+  relayHub: "0x1F3d1C33977957EA41bEdFDcBf7fF64Fd3A3985e",
+  stakeManager: "0xd494924348e91433218506bc0ed02b3a2d83ef0e",
+  forwarder: "0xd9c1a99e9263B98F3f633a9f1A201FA0AFC2A1c2",
   gasPrice: 20000000000, // 20 Gwei
 };
```
Comment (simboonlong) [inline src/config/config.ts:27]: just curious, where did u get these values from

Comment (isaackps) [reply src/config/config.ts:27]: from. here: https://docs-v2.opengsn.org/contracts/addresses201.html#goerli-testnet

Comment (simboonlong) [reply src/config/config.ts:27]: mm not familiar with gsn portion. will approve first, but if @superical has any comments about supporting gsn in future can chip in

**Review by simboonlong — APPROVED** [review]

File: src/config/config.ts (hunk 1 above)
Comment (superical) [reply src/config/config.ts:27]: Are we really utilising GSN in creator? I remember asking something similar in https://github.com/TradeTrust/document-creator-website/pull/262#discussion_r994157739 when I first saw that GSN bit. @isaackps has clarified that it was previously added but not used. Should we remove it if we aren’t using it anymore? But of course if there are still plans to use it then it makes sense to keep it.

Comment (isaackps) [reply src/config/config.ts:27]: @superical ya for this we need to discuss with PO. So if wanna remove it I think we do it in another branch ba.

**Pull request merged**